pkg/debug/scripts/collect-info.sh

```diff
@@ -16,9 +16,10 @@ VERSION=26
 PKG_DEPS="procps tar dmidecode iptables dhcpcd"
 
 DATE=$(date "+%Y-%m-%d-%H-%M-%S")
-INFO_DIR="eve-info-v$VERSION-$DATE"
-TARBALL_FILE="/persist/$INFO_DIR.tar.gz"
+INFO_DIR_SUFFIX="eve-info/v$VERSION-$DATE"
+TARBALL_FILE="/persist/$INFO_DIR_SUFFIX.tar.gz"
 SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+mkdir -p "/persist/$INFO_DIR_SUFFIX"
```

Comment:
Oops. I missed this during the review. By mistake, I decided this directory was created on line 163.
As for me, it would be more natural to see this near somewhere there (around line 163), where the other directories are created. But as long as it works, it's also fine.

Reply:
You probably missed it because the mkdir was not there when I initially generated the PR ;-)

The other mkdirs are for the TMP_DIR so it didn't quite fit there. 

But I don't quite understand the different use of directories in this script. We have a the above /persist/$INFO_DIR_SUFFIX but when the script has completed that directory exists and is empty. Hence it seems like a temporary directory as well.